contrib/opencensus-ext-azure/opencensus/ext/azure/metrics_exporter/__init__.py

```diff
@@ -239,11 +245,57 @@ def _transmit_without_retry(self, envelopes):
                 text,
             )
 
+    def enable_standard_metrics(self):
+        # Sets up stats related objects to begin
+        # recording standard metrics
+        stats_ = stats.stats
```

Comment:
Curious, why not use `stats_module.stats` for consistency?

Reply:
I'll use stats_module.stats for consistency.